src/lib/y2storage/simple_etc_crypttab_entry.rb

```diff
@@ -0,0 +1,112 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+
+module Y2Storage
+  # Information about one entry in crypttab
+  #
+  # This is a wrapper for Storage::SimpleEtcCrypttabEntry
+  class SimpleEtcCrypttabEntry
+    include StorageClassWrapper
+    wrap_class Storage::SimpleEtcCrypttabEntry
+
+    # @!method name
+    #   @return [String] name of the resulting encrypted block device
+    storage_forward :name
+
+    # @!method device
+    #   @return [String] path to the underlying block device
+    storage_forward :device
+
+    # @!method password
+    #   @return [String]
+    storage_forward :password
+
+    # @!method crypt_options
+    #   @return [Array<String>]
+    storage_forward :crypt_options
+
+    # Device for the crypttab entry
```

Comment:
I would be more explicit about the fact that this method returns the device on top of which the `Encryption` device is defined, not the `Encryption` block device itself. I understand it's a more useful approach (for example in cases in which the `Encryption` device is not there in the devicegraph), but it's kind of weird.